proto/messages.proto

```diff
@@ -121,14 +121,15 @@ message AgentConnected {
 message AppDeployment {
   string id = 1;
   string title = 2;
-  string sha = 3;
+  string sha = 3 [deprecated = true];
```

Comment:
I don't mind having this information, it's orthogonal to the version change :D (unless there's a reason I'm missing)

Reply:
Agreed. We can keep it!